src/utils/supabase/queries.ts

```
@@ -0,0 +1,164 @@
+import { SupabaseClient } from '@supabase/supabase-js';
+import { cache } from 'react';
+
+// Users and authentication
+export const getUser = cache(async (supabase: SupabaseClient) => {
+  const {
+    data: { user }
+  } = await supabase.auth.getUser();
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)